chiselFrontend/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -180,6 +208,9 @@ private[chisel3] class DynamicContext() {
   val components = ArrayBuffer[Component]()
   val annotations = ArrayBuffer[ChiselAnnotation]()
   var currentModule: Option[BaseModule] = None
+  // Contains a mapping from a elaborated module to their aspect
+  // Set by [[ModuleAspect]]
+  val aspectModule: mutable.HashMap[BaseModule, BaseModule] = mutable.HashMap.empty[BaseModule, BaseModule]
```

Comment:
So you can't have multiple aspects. This would only happen if you were aspecting a ModuleAspect, which you can't do given its an internal API. Basically this is only for the InjectingAspect, not Aspects generally.

Reply:
Okay, that sounds good!